### DISCUSSION TITLE
Extent coordinates order

### DISCUSSION THREAD

**Discussion by olivier5741** [komoot/photon#708] [Q&A]
Hello :),

The _extent_ coordinates order was puzzling me.

```
"extent": [
    4.3690256,
    50.8007758,
    4.3693617,
    50.8005888
]
```

Is Photon following GeoJSON format ?

Wouldn't it be this ? (bottom left corner then top right corner ?)
https://geojson.org/geojson-spec.html#bounding-boxes
Another info I found interesting : https://www.mongodb.com/docs/manual/reference/operator/query/box/

```
"extent": [
    4.3690256,
    50.8005888, // swapped
    4.3693617,
    50.8007758, // swapped
]
```

Cheers ;),
Olivier

**Comment by jershell** [comment]
I also encountered this feature, can anyone explain me what is wrong with the order in the extent field?

**Reply by lonvia** [reply]
There are twelve different ways to specify a bbox. Photon uses upper-left then lower-right coordinate, which corresponds to the order in [OpenSearch's envelope data type](https://docs.opensearch.org/latest/mappings/supported-field-types/geo-shape/#envelope).
